poll/public/js/poll.js

```diff
@@ -18,7 +18,23 @@ function PollUtil (runtime, element, pollType) {
         }
 
         // Make gettext available in Handlebars templates
-        Handlebars.registerHelper('i18n', function(str) { return gettext(str); });
+        Handlebars.registerHelper('i18n', function(str, options) {
+            options = options.hash;
+            str = gettext(str);
+            if (options) {
+                str = str.supplant(options)
```

Comment:
@bradenmacdonald I was looking into strings [here](https://github.com/open-craft/xblock-poll/blob/fa0aa99/poll/public/handlebars/poll_results.handlebars#L30) and trying to learn more about handlebar custom helpers. The strings contain some dynamic content {total} that need to handle by our `i18n` helper method. Please share your thoughts on this implementation, Sorry these `handlebars` are totally new for me. Your suggestions are appreciated. 
Thanks.